### PR TITLE
add an auxilary array for x86 reg to greatly speedup X86_insn_reg_intel

### PR DESCRIPTION
**Pull request by crowell**

### CODE REVIEW

**Comment by danghvu** [comment]
Do we need qsort ? If it's small enough and execute once, I prefer a simple sort.

**Comment by crowell** [comment]
@danghvu really? you want it to be bubble sort? quicksort is pretty simple.

**Comment by danghvu** [comment]
Alright, maybe just for me.. It looks overkill: you add two more functions and then having recursion.

**Comment by crowell** [comment]
edit: how about just stdlib's qsort

**Comment by crowell** [comment]
current one segfaults dont merge as is, i'll fixup tonight

**Comment by crowell** [comment]
ok, works now

**Comment by danghvu** [comment]
LGTM cc: @aquynh

**Comment by radare** [comment]
lgtm too. any plan to merge this?

File: arch/X86/X86Mapping.c [hunk 1]
```diff
@@ -2751,19 +2752,46 @@ static struct insn_reg2 insn_regs_intel2[] = {
 	{ X86_INVLPGA64, X86_REG_RAX, X86_REG_ECX, CS_AC_READ, CS_AC_READ },
 };
 
+struct insn_reg insn_regs_intel_sorted [ARR_SIZE(insn_regs_intel)];
```
Comment (aquynh) [inline arch/X86/X86Mapping.c]: this struct should be static

**Comment by aquynh** [comment]
merged, thanks!

**Pull request merged**